test/performance/switzerland/build-config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "osmDefaults": {
+    "osmTagMapping": "germany",
+    "timeZone": "Europe/Berlin"
+  },
+  "staticBikeParkAndRide": true,
+  "discardMinTransferTimes": true,
+  "transitModelTimeZone": "Europe/Berlin",
+  "transitFeeds": [
+    {
+      "type": "gtfs",
+      "source": "https://opentransportdata.swiss/en/dataset/timetable-2022-gtfs2020/resource_permalink/gtfs_fp2022_2022-11-02_04-15.zip",
```

Comment:
Do you want me to copy these files so the test is always using the exact same input?

Reply:
Check out https://otp-performance.leonard.io/data/switzerland/

Reply:
Nice, yes it would be more consistent. The gtfs shouldn't change however OSM data will evolve indeed. I'll update PR with updated URLs.

Reply:
Btw random idea, it would be cool to have tests for the same area with static data and dynamic data. Then we could also identify what types of changes in data affect routing performance. This is just an idea and should not be implemented in the scope of this pr.

Reply:
That would not be terribly hard to do. As long as it's clearly communicated that the input data changes and the test is no longer done under "scientific conditions" I'm ok with it.

Reply:
For it to be useful, we would also need to have older versions of the data available for comparison, if there are some performance changes we would like to investigate.

Reply:
I think a new performance test designed for this would be nice.

If you use some gtfs-rt/siri "dump" for the tests, you could have it predictable/scientific again.

And you could have different dump with delays, or cancellations/injections, etc. But I don't expect the performance impact to be very different depending on the dynamic data. The frequency of the update might have an impact and could be interesting to test too (one single update before test or frequent concurrent trip update applied).